feature-utils/poly-look/src/react-components/buttons/polyButton.jsx

```diff
@@ -8,15 +8,16 @@ import "./polyButton.css";
  * @param {Object} props
  * @param {string} [props.label] - Button text
  * @param {boolean} [props.centered] - If the button should center itself.
- * @param {string} [props.type = "filled"] - Type of the button
+ * @param {string} [props.type = "primary"] - Type of the button
  */
 
 const types = {
-  filled: "filled",
```

Comment:
I would revert this change. Ultimately the type is not about the colors but whether if you want a filled or outlined button.
For the colors, there are multiple ways to override the defaults. If it doesn't match the theme configuration, we can always pass a css class to the component or maybe use locally scooped css variables to override the colors.